champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/brute/data/BlockTossObject.java

```diff
@@ -92,7 +97,7 @@ public void spawn(double size) {
         // Create block displays
         displayBlocks = new ArrayList<>();
         final World world = location.getWorld();
-        for (int i = 0; i < 10; i++) { // 10 block displays
+        for (int i = 0; i < 30; i++) { // 10 block displays
```

Comment:
This can cause some unnecessary overhead, are you sure creating 30 display blocks is necessary?

Reply:
can get away with 25